src/main/kotlin/net/irisshaders/lilybot/extensions/events/ThreadInviter.kt

```diff
@@ -73,6 +79,23 @@ class ThreadInviter : Extension() {
 				config.supportTeam ?: return@action
 				config.supportChannel ?: return@action
 
+				if (!guildFor(event)!!.getChannelOf<TextChannel>(config.supportChannel)
+						.getEffectivePermissions(this@event.kord.selfId).contains(
+							Permissions(Permission.SendMessages, Permission.CreatePublicThreads)
+						)
+				) {
+					// We don't check the configured action log permissions here, since it won't have been set properly
```

Comment:
Permissions could've changed since the config was set, we should definitely check.